emmanuel_ogbeide/ASSIGNMENTS/cointosses.py

```diff
@@ -0,0 +1,14 @@
+import random
+
+def cointosses():
+    count1 = 0
+    count2 = 0
+    for i in range(1,5001):
+        rando = random.randint(1,2)
+        if rando == 1:
+            count1 = count1 + 1
+            print "Attempt #" + str(i) + ": Throwing a coin...  It's a head! ... Got " + str(count1) + " head(s) so far"
+        else:
+            count2 = count2 + 1
+            print "Attempt #" + str(i) + ": Throwing a coin...  It's a tail! ... Got " + str(count2) + " tail(s) so far"
```

Comment:
Anytime you see something repeated, try and find a way to condense it down.  In this case you have almost exactly the same print statements.  You can set your variables inside the if/else blocks and then just make one print statement at the end outside of the if/else (you're output is a little different than the assignment asks for too)